src/components/Layout/index.jsx

```diff
@@ -1,9 +1,20 @@
 import { Footer } from "../Footer/index.jsx";
 import { NavBar } from "../NavBar/index.jsx";
+import React, { useContext } from "react";
+import { Context } from "../../App.jsx";
+import { VerticalContainer } from "../../shared/styledComponents/verticalContainer.styled.js";
+import { Snackbar } from "@mui/material";
 
 export function Layout({ children }) {
+  const contextIsError = useContext(Context)[0];
```

Comment:
Please don't use index on the context. This is not readable at all. Please refer to the documentation https://react.dev/reference/react/createContext

Reply:
done